packages/esm-hiv-app/src/views/visits/tabs/clinical-visit-tab.component.tsx

```diff
@@ -14,79 +14,82 @@ interface ClinicalVisitListProps {
   patientUuid: string;
 }
 
-const columns: EncounterListColumn[] = [
-  {
-    key: 'visitDate',
-    header: 'Visit Date',
-    getValue: (encounter) => {
-      return getObsFromEncounter(encounter, dateOfEncounterConcept, true);
-    },
-    link: {
-      getUrl: (encounter) => encounter.url,
-      handleNavigate: (encounter) => {
-        encounter.launchFormActions?.viewEncounter();
-      },
-    },
-  },
-  {
-    key: 'visitType',
-    header: 'Visit Type',
-    getValue: (encounter) => {
-      return getObsFromEncounter(encounter, visitTypeConcept);
-    },
-  },
-  {
-    key: 'screeningOutcome',
-    header: 'TB Screening Outcome',
-    getValue: (encounter) => {
-      return getObsFromEncounter(encounter, tbScreeningOutcome);
-    },
-  },
-  {
-    key: 'nextAppointmentDate',
-    header: 'Next Appointment Date',
-    getValue: (encounter) => {
-      return getObsFromEncounter(encounter, returnVisitDateConcept, true);
-    },
-  },
-  {
-    key: 'appointmentReason',
-    header: 'Appointment Reason',
-    getValue: (encounter) => {
-      return '--';
-    },
-  },
-  {
-    key: 'actions',
-    header: 'Actions',
-    getValue: (encounter) => {
-      const baseActions = [
-        {
-          form: { name: 'clinical_visit', package: 'hiv' },
-          encounterUuid: encounter.uuid,
-          intent: '*',
-          label: 'View Details',
-          mode: 'view',
-        },
-        {
-          form: { name: 'clinical_visit', package: 'hiv' },
-          encounterUuid: encounter.uuid,
-          intent: '*',
-          label: 'Edit Form',
-          mode: 'edit',
-        },
-      ];
-      return baseActions;
-    },
-  },
-];
-
 const ClinicalVisitList: React.FC<ClinicalVisitListProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
 
   const headerTitle = t('clinicalVisit', 'Clinical Visit');
   const displayText = t('clinicalVisit', 'Clinical Visit');
 
+  const columns: EncounterListColumn[] = useMemo(
+    () => [
+      {
+        key: 'clinicalVisitDate',
+        header: t('visitDate', 'Visit Date'),
+        getValue: (encounter) => {
+          return getObsFromEncounter(encounter, dateOfEncounterConcept, true);
+        },
+        link: {
+          getUrl: (encounter) => encounter.url,
+          handleNavigate: (encounter) => {
+            encounter.launchFormActions?.viewEncounter();
+          },
+        },
+      },
+      {
+        key: 'clinicalVisitType',
+        header: t('visitType', 'Visit Type'),
+        getValue: (encounter) => {
+          return getObsFromEncounter(encounter, visitTypeConcept);
+        },
+      },
+      {
+        key: 'clinicalScreeningOutcome',
+        header: t('screeningOutcome', 'TB Screening Outcome'),
```

Comment:
this needs to be aligned

Reply:
@pirupius I don't understand your comment here

Reply:
The text is 'TB Screening Outcome' so the translation key should match that i.e `tbScreeningOutcome`